llvm/lib/DebugInfo/GSYM/DwarfTransformer.cpp

```diff
@@ -612,11 +617,16 @@ Error DwarfTransformer::convert(uint32_t NumThreads, OutputAggregator &Out) {
       DWARFDie Die = getDie(*CU);
       if (Die) {
         CUInfo CUI(DICtx, dyn_cast<DWARFCompileUnit>(CU.get()));
-        pool.async([this, CUI, &LogMutex, &Out, Die]() mutable {
+        pool.async([this, CUI, &CU, &LogMutex, &Out, Die]() mutable {
```

Comment:
If "CU" is a pointer, do we need to capture a reference? Could this just be:
```
pool.async([this, CUI, CU, &LogMutex, &Out, Die]() mutable {
```
Or does capture make `CU` into a `const *`?

Reply:
Not a pointer: a unique_ptr, so only capture by reference.